packages/shader-lab/src/parser/ShaderTargetParser.ts

```
@@ -103,8 +114,19 @@
+        // #if _EDITOR
+        this.sematicAnalyzer.errors.push(
+          new GSError(
+            GSErrorName.CompilationError,
+            `Unexpected token ${token.lexeme}`,
+            token.location,
+            ShaderLab._processingPassText
+          )
+        );
         return null;
+        // #else
+        throw new Error(`Unexpected token ${token.lexeme}`);
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)